src/common/utils.js

```diff
@@ -49,29 +49,34 @@ Class.subClass = function( props )
 };
 
 // An enhanced DataView
-// Accepts an ArrayBuffer, typed array, or a length number
-function MemoryView( buffer, byteOffset = 0, byteLength )
```

Comment:
(Removed used of ES6 default parameter, replaced with |= 0 below.)

Reply:
I've gradually started introducing ES6 features as I change code, but this is so minor it doesn't matter either way.